.azure/templates/jobs/build/build_strimzi.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         displayName: "Build Java code"
         env:
           MVN_ARGS: "-DskipTests -e -V -B"
+          RUN_ON_CI: "true"
```

Comment:
TBH, I would avoid this and follow what we have for Maven with `CURL_PARAM` option that is set to `-s` rather than a flag like this. That gives us much more flexibility and consistency.

Reply:
While I see your point, curl could be used for other objectives (in the future) and with different parameters. Using a specific env var would tight to a specific list.
Also right now the curl commands we are referring to are ...

`curl -L"${curl_silent}" -o "${local_path}" -w %{http_code} "${cdn_url}"`
and
`curl -L"${curl_silent}" -o "${local_path}" -w %{http_code} "${archive_url}"`

so even in this case different parameters with variables as well (see `-o`)
Which curl parameters would be involved in the `CURL_PARAM`, only `-Ls` in this case? does it make sense just for some of them?

Reply:
I guess `-L` has to always be there. So it is `-s` you need to pass.

Reply:
PS: `CURL_ARGS` probably makes more sense then `CURL_PARAM`. Not sure why I used the wrong one initiially.

Reply:
> While I see your point, curl could be used for other objectives (in the future) and with different parameters. Using a specific env var would tight to a specific list.

Well, this is the same with Maven as well. The `MVN_ARGS` has different values in different places. That is the flexibility it gives you that a single boolean flag cannot.